Tests/conf.json

```diff
@@ -5460,4 +5460,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
please undo this change (added a newline in the end of the file)

Reply:
done.